aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -0,0 +1,328 @@
+.. _web-api-help-pages-using-swagger:
+
+ASP.NET Web API Help Pages using Swagger
+========================================
+
+By `Shayne Boyer`_
+
+Understanding the various methods of an API can be a challenge for a developer when building a consuming application. 
+
+Generating good documentation and help pages as a part of your Web API using `Swagger <a href="http://swagger.io/">`_ with the .NET Core implementation `Swashbuckle <a href="https://github.com/domaindrivendev/Ahoy">`_ is as easy as adding a couple of NuGet packages and modifying the Startup.cs.
+
+This tutorial builds on the sample on :doc:`first-web-api`. If you'd like to follow along, download the sample at https://github.com/aspnet/Docs/tree/master/aspnet/tutorials/first-web-api/sample.
+
+.. contents:: Sections:
+  :local:
+  :depth: 2
+
+Getting Started
+---------------
+There are two core components to Swashbuckle 
+
+- *Swashbuckle.SwaggerGen* : provides the functionality to scaffold your Web API and generate JSON Swagger documents that describe the objects, methods, return types, etc. 
+- *Swashbuckle.SwaggerUI* : an embedded version of the Swagger UI tool which uses the above documents for a rich customizable experience for describing the Web API functionality and includes built in test harness capabilities for the public methods.
+
+NuGet Packages
+--------------
+You can add Swashbuckle with any of the following approaches:
+
+- From the Package Manager Console: 
+
+.. code-block:: bash
+
+    Install=Package Swashbuckle -Pre
+
+- Add Swashbuckle to project.json:
+
+.. code-block:: javascript
+
+    "Swashbuckle": "6.0.0-"
+
+- In Visual Studio:
+	- Right click your project in Solution Explorer > Manage NuGet Packages 
+	- Enter Swashbuckle in the search box
+	- Check "Include prerelease" 
+	- Set the Package source to nuget.org 
+	- Tap the Swashbuckle package and then tap Install 
+
+
+
+Add and configure Swagger to the middleware
+-------------------------------------------
+
+Add SwaggerGen to the services collection in the Configure method, and in the ConfigureServices method, enable the middleware for serving generated JSON document and the SwaggerUI. 
+
+.. literalinclude:: web-api-help-pages-using-swagger/sample/src/TodoApi/Startup.cs
+    :language: c#
+    :start-after: snippet_Configure
+    :end-before: #endregion
+    :dedent: 8
+    :emphasize-lines: 13,44,47
+
```

Comment:
Your startup file is alot different than the one generated by the templates. You've got 
```
 app.UseStaticFiles();
```
which you normally don't want for a Web API project.